### ISSUE TITLE
German Translation is missing with 4.4.20 after pressing "stop".

### ISSUE THREAD

**Issue by MarkusKress** [QIDITECH/QIDI_Q1_Pro#48]
After stopping the print, a question pops up, but in Chinese. I think it's yes and no, but I don't know.

**Comment by CChen616** [comment]
Yes, the translation for the emergency stop confirmation page is missing.

The patched update package has been uploaded to the release page.

**Comment by CChen616** [comment]
The issue has been resolved in the 4.4.20 patch and subsequent versions. If there are no other concerns, I will close this issue.
